docker-compose.rest-api.dev.yaml

```diff
@@ -0,0 +1,36 @@
+version: '3.0'
+
+services:
+    nginx:
+        image: spryker/php-sdk-api-nginx:${SDK_VERSION:-latest}
+        build:
+            context: .
+            dockerfile: infrastructure/web/nginx.Dockerfile
+        ports:
+            - ${SDK_REST_API_PORT:-80}:80
+        volumes:
+            - ./public:/data/public
+        depends_on:
+            - php-fpm
+        restart: always
```

Comment:
Please replace it with restart: on-failure to prevent restarting loop.
The same for the prod env file.

Reply:
It doesn't prevent loop. But if the docker be restarted the services will not be upped.